src/server/services/data-transform.service.test.js

```diff
@@ -679,6 +679,96 @@ describe("data-transform.service transformAnswersForSubmit()", () => {
     });
   });
 
+  describe("when opening hours are empty strings", () => {
```

Comment:
should we be catching whitespace only strings?

Reply:
Validation service ensures whitespace only strings are not allowed